rest/admin_api.go

```diff
@@ -105,12 +105,15 @@ func (h *handler) handleCreateDB() error {
 		// store the cas in the loaded config after a successful insert
 		h.server.dbConfigs[dbName].cas = cas
 	} else {
-		if err := config.setup(dbName, h.server.config.Bootstrap, nil); err != nil {
+		if err := config.setup(dbName, BootstrapConfig{}, nil); err != nil {
 			return err
 		}
 
 		// load database in-memory for non-persistent nodes
-		if _, err := h.server.AddDatabaseFromConfig(DatabaseConfig{DbConfig: *config}); err != nil {
+		if _, err := h.server.AddDatabaseFromConfigFailFast(DatabaseConfig{DbConfig: *config}); err != nil {
+			if errors.Is(err, base.ErrAuthError) {
+				return base.HTTPErrorf(http.StatusForbidden, "auth failure accessing provided bucket using database credentials: %s", base.Metadata(bucket))
```

Comment:
As discussed don't think we want `bucket` here given the possible change to collections in the future. Might be better to just tie wording to database. 

Even without that might want to move where the bucket name is. The error would be:
`auth failure accessing provided bucket using database credentials: bucketName` which doesn't make sense. At minimum something like:
`auth failure accessing bucket %s using provided credentials`.

But again, try to tie it to database I think